### PR TITLE
(#1190) Foundations Color Page

### LINKED ISSUE #1190: Foundations subpage: Color
Update the Color page within the Foundations section of the documentation site with the content needed for NCIDS for All pilot launch.

### Tasks
- [x] Create content in [shared document](https://nih.sharepoint.com/:w:/r/sites/NCI-OCPLNewCgovUXDesignSystem/_layouts/15/Doc.aspx?sourcedoc=%7BF1F53375-736E-4AAC-8714-3CFE9B78CFDE%7D&file=Color.docx&wdLOR=c9CB93DB6-A176-5344-B038-C91CE341093B&action=default&mobileredirect=true)
    - [x] IA responsibility:
    - [x] Dev responsibility:
    - [x] Design responsibility:
- [x] Review content in shared document
- [x] Create images in shared folder: `<insert link to shared folder>`
- [x] Convert content and images to markdown
- [x] Add to documentation site 
- [x] Add content from the Foundations/Appearance/Color page (see original [shared doc here](https://nih.sharepoint.com/:w:/r/sites/NCI-OCPLNewCgovUXDesignSystem/_layouts/15/Doc.aspx?sourcedoc=%7B8536D727-9354-47B3-8780-6AC37CB96F69%7D&file=Appearance%20-%20Color.docx&action=default&mobileredirect=true)) to this page
- [x] Re-review content in shared document
- [x] Update markdown
- [x] Update documentation site
  - [x] Remove Foundations/Appearance/Color
  - [x] Remove Foundations/Appearance/State Color Tokens
  - [x] Remove Foundations/Appearance/System Color Tokens
  - [x] Remove Foundations/Appearance/Theme Color Tokens

### Resources
[Doc Site Content Inventory](https://nih.sharepoint.com/:x:/r/sites/NCI-OCPLNewCgovUXDesignSystem/Shared%20Documents/General/Details%20-%20NCIDS%20Pilot/Documentation%20Site/Doc%20Site%20Content%20Inventory.xlsx?d=wff83f29cd1554e99892fd963404c69a1&csf=1&web=1&e=QZgQLq)

### PR DESCRIPTION
**Pull request by karlikpj**
Closes #1190

### CODE REVIEW

**Review by olitharp-nci — APPROVED** [review]

**Review by bennettcc — APPROVED** [review]

**Comment by bennettcc** [comment]
Approved on behalf of product - they reviewed and approved on 9/20

**Comment by bennettcc** [comment]
This was reverified by product on 10/4 - passes their testing

File: docs/content/foundations/color.mdx
Comment (olitharp-nci) [inline docs/content/foundations/color.mdx:1]: make sure this file passes a11y

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

File: docs/content/foundations/color.mdx
Comment (olitharp-nci) [reply docs/content/foundations/color.mdx:1]: Screenshot 2023-10-10 at 10 01 22 AM

**Review by olitharp-nci — CHANGES_REQUESTED** [review]

**Comment by sarinapadilla** [comment]
@karlikpj we discussed removing the Foundations > Appearance > Color page and only having the Foundations > Color page. If we could include removing that and the removal of the Color link on the Appearance landing page on this PR, that would be great!

**Review by andyvanavery31 — APPROVED** [review]
Passes Product review.

**Review by olitharp-nci — APPROVED** [review]

**Pull request merged**